sdk/src/cache/cache_client.rs

```diff
@@ -1051,10 +1054,10 @@ impl CacheClient {
     /// * `sorted_set_name` - The name of the sorted set to add an element to.
     /// * `order` - The order to sort the elements by. [SortedSetOrder::Ascending] or [SortedSetOrder::Descending].
     /// * `start_rank` - The rank of the first element to fetch. Defaults to 0. This rank is
-    /// inclusive, i.e. the element at this rank will be fetched.
+    ///   inclusive, i.e. the element at this rank will be fetched.
```

Comment:
apologies for these docstring whitespace changes, they were necessary because apparently some clippy rules have been updated since the last time we made a commit to this repo.